recipe/build.sh

```diff
@@ -2,27 +2,33 @@ export CMAKE_LIBRARY_PATH=$PREFIX/lib:$PREFIX/include:$CMAKE_LIBRARY_PATH
 export CMAKE_PREFIX_PATH=$PREFIX
 export PATH=$PREFIX/bin:$PATH
 
-export CUDA_ARCH_LIST="-gencode arch=compute_37,code=sm_37 -gencode arch=compute_50,code=sm_50 -gencode arch=compute_60,code=sm_60 -gencode arch=compute_70,code=sm_70"
+CUDA_ARCH_LIST="-gencode arch=compute_37,code=sm_37"
+CUDA_ARCH_LIST="${CUDA_ARCH_LIST} -gencode arch=compute_50,code=sm_50"
+CUDA_ARCH_LIST="${CUDA_ARCH_LIST} -gencode arch=compute_60,code=sm_60"
+CUDA_ARCH_LIST="${CUDA_ARCH_LIST} -gencode arch=compute_70,code=sm_70"
 
 if [[ "$cuda_compiler_version" == "11.1" || "$cuda_compiler_version" == "11.2" ]]; then
-  CUDA_ARCH_LIST="$CUDA_ARCH_LIST -gencode arch=compute_80,code=sm_80 -gencode arch=compute_86,code=sm_86"
+  CUDA_ARCH_LIST="${CUDA_ARCH_LIST} -gencode arch=compute_80,code=sm_80"
+  CUDA_ARCH_LIST="${CUDA_ARCH_LIST} -gencode arch=compute_86,code=sm_86"
 elif [[ "$cuda_compiler_version" == "11.0" ]]; then
-  CUDA_ARCH_LIST="$CUDA_ARCH_LIST -gencode arch=compute_80,code=sm_80"
+  CUDA_ARCH_LIST="${CUDA_ARCH_LIST} -gencode arch=compute_80,code=sm_80"
 fi
 
-# std=c++11 is required to compile some .cu files
-CPPFLAGS="${CPPFLAGS//-std=c++17/-std=c++14}"
-CXXFLAGS="${CXXFLAGS//-std=c++17/-std=c++14}"
+CUDA_NVCC_FLAGS="${CUDA_NVCC_FLAGS} -Xfatbin -compress-all"
```

Comment:
For example, it looks like you're trying to add some new compile flags, but I'm not sure that these flags exist for NVCC 11.2

```suggestion
CUDA_NVCC_FLAGS="${CUDA_NVCC_FLAGS} --fatbin"
```

https://docs.nvidia.com/cuda/archive/11.2.2/cuda-compiler-driver-nvcc/index.html#options-for-specifying-compilation-phase-fatbin